.github/workflows/ci.yml

```diff
@@ -7,18 +7,17 @@ env:
   BUNDLE_WITHOUT: release
 
 jobs:
-  # rubocop is currently not included in Gemfile
-  # rubocop:
-  #   runs-on: ubuntu-latest
-  #   steps:
-  #     - uses: actions/checkout@v4
-  #     - name: Install Ruby 3.1
-  #       uses: ruby/setup-ruby@v1
-  #       with:
-  #         ruby-version: "3.1"
-  #         bundler-cache: true
-  #     - name: Run rubocop
-  #       run: bundle exec rake rubocop
+  rubocop:
```

Comment:
can you adopt the pattern we've here: https://github.com/voxpupuli/voxpupuli-release/blob/master/.github/workflows/test.yml#L47

that provides us a dummy job that finishes when all other jobs are finished. that enables us to enforce this dummy job for PRs. Then we don't need to make all individual CI jobs mandatory, which is a pain when we add new ruby version support.

Reply:
@bastelfreak is https://github.com/voxpupuli/beaker-google/pull/50/commits/4569100df513257097e3f243217c3d6b654be265 sufficient?

Reply:
yes!